release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2647,6 +2647,38 @@ To avoid this issue, you can enable confidential VMs on an existing cluster by u
+(link:https://issues.redhat.com/browse/RHEL-9148[*RHEL-9148*])
+
+* Low-latency applications that rely on high-resolution timers to wake up their threads might experience higher wake up latencies than expected. Although the expected wake up latency is under 20μs, latencies exceeding this can occasionally be seen when running the cyclictest tool for long durations (24 hours or more). Testing has shown that wake up latencies are under 20μs for over 99.999999% of the samples.
+(link:https://issues.redhat.com/browse/RHELPLAN-138733[*RHELPLAN-138733*])
+
+* The GNSS module in an Intel Westport Channel e810 NIC that is configured as a grandmaster clock (T-GM) can report the GPS `FIX` state and the GNSS offset between the GNSS module and the GNSS constellation satellites.
++
+The current T-GM implementation does not use the `ubxtool` CLI to probe the `ublox` module for reading the GNSS offset and GPS `FIX` values.
+Instead, it uses `gpsd` to read the GPS `FIX` information.
+This is because the current implementation of the `ubxtool` CLI takes 2 seconds to receive a response, and with every call, it increases CPU usage threefold.
+(link:https://issues.redhat.com/browse/OCPBUGS-17422[*OCPBUGS-17422*])
+
+* When GNSS is lost, the driver's DPLL state can transition to `unlocked` or it can enter a holdover state. Currently, the driver transitions to the `unlocked` state. Upstream efforts are currently implementing second state handling and a mechanism to configure which state machine handling is desired.
+(link:https://issues.redhat.com/browse/RHELPLAN-164754[*RHELPLAN-164754*])
+
+* The DPLL subsystem and DPLL support is not currently enabled in the Intel Westport Channel e810 NIC ice driver. The feature request is being tracked under link:https://issues.redhat.com/browse/RHELPLAN-165955[RHELPLAN-165955].
+(link:https://issues.redhat.com/browse/RHELPLAN-165955[*RHELPLAN-165955*])
+
+* The current grandmaster clock (T-GM) implementation has a single NMEA sentence generator sourced from the GNSS without a backup NMEA sentence generator.
+If NMEA sentences are lost on their way to the e810 NIC, the T-GM cannot synchronize the devices in the network synchronization chain and the PTP Operator reports an error.
+A proposed fix is to report a `FREERUN` event when the NMEA string is lost.
+(link:https://issues.redhat.com/browse/OCPBUGS-19838[*OCPBUGS-19838*])
+
+* Currently, due to an issue with CPU resource configurations in containers that use `crun` runtimes, low-latency workloads can feature performance degradation. As a workaround, use `runc` container runtimes. Although `runc` container runtimes have lower performance during container startup, shutdown operations, and `exec` probes, the `crun` and `runc` container runtimes are functionally the same. It is anticipated that an upcoming z-stream release will include a fix for this issue. 
```

Comment:
`due to differences in setting up a container's cgroup hierarchy, containers that use the `crun` OCI runtime along with a PerformaceProfile encounter performance degradation.`